src/black/__init__.py

```diff
@@ -311,8 +311,16 @@ def find_pyproject_toml(path_search_start: Tuple[str, ...]) -> Optional[str]:
     if path_pyproject_toml.is_file():
         return str(path_pyproject_toml)
 
-    path_user_pyproject_toml = find_user_pyproject_toml()
-    return str(path_user_pyproject_toml) if path_user_pyproject_toml.is_file() else None
+    try:
+        path_user_pyproject_toml = find_user_pyproject_toml()
+        return (
+            str(path_user_pyproject_toml)
+            if path_user_pyproject_toml.is_file()
+            else None
+        )
+    except PermissionError:
+        # We do not have access to the user-level config directory, so ignore it.
```

Comment:
Lets add some logging (atleast @ debug level) so we at least tell the user why their config is not being picked up.
- Could save hours of debugging for someone one day
- Silent swallowing at debug level is evil

Reply:
Good point. I made it just unconditionally print for now. Maybe we'd want to do it only if `-v` is passed, but that's hard because this code gets executed during argument parsing.